### ISSUE TITLE
Implement "addressForCoordinate" from "react-native-maps"

### ISSUE THREAD

**Issue by danieldanielecki** [tamotam-com/tamotam-app#1]
The `addressForCoordinate` is essential for `tamotam` application to get coordinates of dragged positions to make REST API calls in that scenario and, therefore, display pins of events on the map.

Follow-up issue [#3775](https://github.com/react-native-maps/react-native-maps/issues/3775) to track once it is merged in the possibly next release.

**Comment by danieldanielecki** [comment]
It has been deleted for cleaning up purpose.

**Comment by danieldanielecki** [comment]
We'll use this only when/if we'll have our own API where we'll build the functionality to unify all events having `countryCode`.
